packages/ui/src/utils/connectors/state.ts

```diff
@@ -0,0 +1,34 @@
+import { generateRandomString } from '..';
```

Comment:
📌 This code existed already, split the files down into base, social and blockchain